contracts/scripts/foundry/InitializeL2ScrollOwner.s.sol

```diff
@@ -0,0 +1,237 @@
+        Ownable(L2_GATEWAY_ROUTER_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_CUSTOM_ERC20_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_ETH_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_STANDARD_ERC20_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_WETH_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_ERC721_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+        Ownable(L2_ERC1155_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+
+        // Ownable(L2_USDC_GATEWAY_PROXY_ADDR).transferOwnership(address(owner));
+
+        Ownable(L2_ETH_RATE_LIMITER_ADDR).transferOwnership(address(owner));
+        AccessControlEnumerable(L2_TOKEN_RATE_LIMITER_ADDR).grantRole(bytes32(0), address(owner));
+        AccessControlEnumerable(L2_TOKEN_RATE_LIMITER_ADDR).revokeRole(bytes32(0), vm.addr(L2_DEPLOYER_PRIVATE_KEY));
+    }
+
+    function grantRoles() internal {
+        owner.grantRole(SECURITY_COUNCIL_NO_DELAY_ROLE, SECURITY_COUNCIL_ADDR);
+        owner.grantRole(SCROLL_MULTISIG_NO_DELAY_ROLE, SCROLL_MULTISIG_ADDR);
+        owner.grantRole(TIMELOCK_1DAY_DELAY_ROLE, L2_1D_TIMELOCK_ADDR);
+        owner.grantRole(TIMELOCK_7DAY_DELAY_ROLE, L2_7D_TIMELOCK_ADDR);
+
+        owner.grantRole(bytes32(0), L2_14D_TIMELOCK_ADDR);
+        owner.revokeRole(bytes32(0), vm.addr(L2_DEPLOYER_PRIVATE_KEY));
+    }
+
+    function configProxyAdmin() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, security council
+        _selectors = new bytes4[](3);
+        _selectors[0] = ProxyAdmin.changeProxyAdmin.selector;
+        _selectors[1] = ProxyAdmin.upgrade.selector;
+        _selectors[2] = ProxyAdmin.upgradeAndCall.selector;
+        owner.updateAccess(L2_PROXY_ADMIN_ADDR, _selectors, SECURITY_COUNCIL_NO_DELAY_ROLE, true);
+    }
+
+    function configL1GasPriceOracle() internal {
+        bytes4[] memory _selectors;
+
+        // no delay, scroll multisig
+        _selectors = new bytes4[](2);
+        _selectors[0] = L1GasPriceOracle.setOverhead.selector;
+        _selectors[1] = L1GasPriceOracle.setScalar.selector;
+        owner.updateAccess(L1_GAS_PRICE_ORACLE_ADDR, _selectors, SCROLL_MULTISIG_NO_DELAY_ROLE, true);
+    }
+
+    function configL2TxFeeVault() internal {
+        bytes4[] memory _selectors;
+
+        // delay 7 day, scroll multisig
+        _selectors = new bytes4[](1);
+        _selectors[0] = L2TxFeeVault.updateMinWithdrawAmount.selector;
+        owner.updateAccess(L2_TX_FEE_VAULT_ADDR, _selectors, TIMELOCK_7DAY_DELAY_ROLE, true);
+    }
+
+    function configWhitelist() internal {
+        bytes4[] memory _selectors;
+
+        // delay 1 day, scroll multisig
+        _selectors = new bytes4[](1);
+        _selectors[0] = Whitelist.updateWhitelistStatus.selector;
+        owner.updateAccess(L2_WHITELIST_ADDR, _selectors, TIMELOCK_1DAY_DELAY_ROLE, true);
```

Comment:
`L1_WHITELIST_ADDR` is missing (both here and in the doc)

Reply:
Fixed in [ab6c986](https://github.com/scroll-tech/scroll/pull/838/commits/ab6c98669af738bbcd2e66e9cdaa17596ad5d492).